themes/prism-atom-dark.css

```diff
@@ -8,7 +8,7 @@ code[class*="language-"],
 pre[class*="language-"] {
 	color: #c5c8c6;
 	text-shadow: 0 1px rgba(0, 0, 0, 0.3);
-    font-family: Inconsolata, Monaco, Consolas, 'Courier New', Courier, monospace;
+	font-family: Inconsolata, Monaco, Consolas, 'Courier New', Courier, monospace;
```

Comment:
Nit, but can we put this in #80, which already has a bunch of whitespace fixes.

Reply:
True.